### PR TITLE
Serializeform.js, all them tests is sure passing

### PR DESCRIPTION
**Pull request by chee**

### CODE REVIEW

File: app/assets/javascripts/lib/utils/serialize_form.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+define([
+  "jquery"
+], function($) {
```
Comment (remybach) [inline app/assets/javascripts/lib/utils/serialize_form.js]: Can this not go on one line and then expand later *if* any extras are required?

Comment (chee) [inline app/assets/javascripts/lib/utils/serialize_form.js]: i suppose, but this way it's more obvious what changed in the diff when something new is added. :o either way is fine with me. YOU WANT ME TO CHANGE IT?

Comment (remybach) [inline app/assets/javascripts/lib/utils/serialize_form.js]: Meh... not a biggie at all.

**Comment by remybach** [comment]
:rocket:

**Pull request merged**